src/lib.ts

```diff
@@ -1,6 +1,10 @@
 // ----- Functions ----- //
 
 const compose = <A, B, C>(f: (_b: B) => C, g: (_a: A) => B) => (a: A): C => f(g(a));
+const pipe = <A, B>(a: A, f: (_a: A) => B): B => f(a);
+const pipe2 = <A, B, C>(a: A, f: (_a: A) => B, g: (_b: B) => C): C => g(f(a));
```

Comment:
Any chance of making `pipe` variadic?

Reply:
Yeah that's an option (no pun intended), [fp-ts](https://github.com/gcanti/fp-ts/blob/1b0ee81826e93ddd86ce20a3c353d349a6994bb6/src/function.ts#L325) does this using the `arguments` length.

I thought about it, but I wasn't sure about the implementation and if it would cause confusion. What do you think?

Reply:
I was expecting the same as well so that might be a good idea